docs/docsite/rst/dev_guide/developing_modules_general_windows.rst

```diff
@@ -38,52 +38,14 @@ When creating a new module there are a few things to keep in mind:
 - Use the full cmdlet name instead of aliases, e.g. ``Remove-Item`` over ``rm``
 - Use named parameters with cmdlets, e.g. ``Remove-Item -Path C:\temp`` over ``Remove-Item C:\temp``
 
-A very basic powershell module template can be found found below:
+A very basic powershell module ``win_environment`` is included below. It demonstrates how to implement check-mode and diff-support, and also shows a warning to the user when a specific condition is met.
 
 .. code-block:: powershell
+.. include:: ../../../../lib/ansible/modules/windows/win_environment.ps1
 
-    #!powershell
-    # This file is part of Ansible
+A slightly more advanced module is `win_uri <https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/windows/win_uri.ps1>` which additionally shows how to use different parameter types (bool, str, int, list, dict, path) and a selection of choices for parameters, how to fail a module and how to handle exceptions.
```

Comment:
Can we shorten the URL text to just be win_uri?

Reply:
I don't understand what you mean. I am not including the content for the more advanced example, so expect people to click through.

Reply:
In markdown down I can go [win_uri](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/windows/win_uri.ps1) instead of putting the full URL there. I would probably let @dharmabumstead comment on it as he knows more about docs than I do.

Reply:
Well, again I think that's related to the RST Preview in Github. The syntax I am using should also do it like this. Both `..include` as well as the Weblink syntax was taken from other chapters where it was generated correctly.